src/reference/environments/envs/cylc-all.yml

```diff
@@ -0,0 +1,8 @@
+name: cylc-|release|
```

Comment:
This gets replaced with the release when the file is read into the docs so we don't have to update it.